drivers/adc-dac/ad5592r/ad5592r-base.h

```diff
@@ -77,11 +77,42 @@ enum ad5592r_registers {
 	AD5592R_REG_RESET		= 0xF,
 };
 
+#define AD5592R_REG_PD_PD_ALL		BIT(10)
+#define AD5592R_REG_PD_PD_ALL_RD(x)		(((x) & AD5592R_REG_PD_PD_ALL) >> 10)
```

Comment:
i don't see the need for a read macro. Where would this be used? If the reason would be just to check if it is set this ca be done with:

```
if (reg & AD5592R_REG_PD_PD_ALL)
{
      prinft("Powerdown is set!\r\n");
}

Reply:
You are correct. I was using them to shift to the LSB but that is not necessary I will remove these